spec/src/main/asciidoc/ch05-metamodel-api.adoc

```diff
@@ -158,15 +161,23 @@ a declaration as follows:
 public static volatile EntityGraph<X> _n;
 ----
 +
+where `n` is the name `"n"` with every character which is not a legal Java identifier character
+replaced with an underscore.
 
 Import statements must be included for the
-needed `jakarta.persistence.metamodel` types as appropriate (e.g.,
-`jakarta.persistence.metamodel.SingularAttribute`,
-`jakarta.persistence.metamodel.CollectionAttribute`,
-`jakarta.persistence.metamodel.SetAttribute`,
-`jakarta.persistence.metamodel.ListAttribute`,
-`jakarta.persistence.metamodel.MapAttribute`) and all classes `X`, `Y`,
-`Z`, `R`, and `K`.
+needed `jakarta.persistence` and `jakarta.persistence.metamodel` types as appropriate (e.g.,
+`jakarta.persistence.EntityGraph`, `jakarta.persistence.TypedQueryReference`,
+`jakarta.persistence.metamodel.SingularAttribute`, `jakarta.persistence.metamodel.CollectionAttribute`,
+`jakarta.persistence.metamodel.SetAttribute`, `jakarta.persistence.metamodel.ListAttribute`,
+`jakarta.persistence.metamodel.MapAttribute`, `jakarta.persistence.metamodel.EntityType`,
+`jakarta.persistence.metamodel.EmbeddableType`, `jakarta.persistence.metamodel.MappedSuperclassType`)
+and all classes `X`, `Y`, `Z`, `R`, and `K`.
```

Comment:
I wonder why we even need this paragraph. I think I would just delete it.

Reply:
I've made this shorter